### DISCUSSION TITLE
Exporters routes

### DISCUSSION THREAD

**Discussion by mateusoliveira43** [dora-metrics/pelorus#892] [General]
Originated from https://github.com/konveyor/pelorus/issues/888

Today, all the endpoints of the Pelorus exporters redirect to `/metrics` endpoint (which is the only one we need, because of Prometheus). It is interesting to have the root `/` redirect to it, because user can easily see it on OpenShift. But redirecting every endpoint (like `/wrong`) seems too much.

What should Pelorus  exporters do?
1. only redirect the root `/` (thinking about the user clicking on the route on OpenShift)
1. just have the `/metrics` endpoint, which is the only one that matters for Prometheus

Other Opinions are welcome as well! :smile:

**Comment by mpryc** [comment]
Actually it's:
Today for the `GET` HTTP method, all the endpoints of the Pelorus exporters redirect to `/`
